src/controller/patientController.ts

```diff
@@ -0,0 +1,133 @@
+import { NextFunction, Request, Response } from "express";
+import * as patientService from "../service/patientService";
+
+/**
+ * Get all patient.
+ * @param {Request} req
+ * @param {Response} res
+ */
+export const getAllPatients = (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  patientService
+    .getAllPatients()
+    .then((data: any) => res.json(data))
```

Comment:
Avoid using any type as much as possible. The essence of typescript lies in limiting types.

Reply:
Sure, I will correct it.